modules/serverless-knative-kafka-broker-metrics.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * serverless/observability/admin-metrics/serverless-admin-metrics.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-knative-kafka-broker-metrics_{context}"]
+= Knative Kafka broker metrics
+
+You can use the following metrics to debug and visualize the performance of Kafka broker.
+
+[cols=5*,options="header"]
+|===
+|Metric name
+|Description
+|Type
+|Tags
+|Unit
+
+|`event_count_1_total{job="kafka-broker-receiver-sm-service", namespace="knative-eventing"}`
+|Number of events received by a Broker
```

Comment:
```suggestion
|Number of events received by a broker
```
Should broker ever be capitalized? If this is valid, then ignore my later comments on the same.

Reply:
You are correct it should be broker.